public/app/plugins/datasource/loki/languageUtils.ts

```diff
@@ -35,6 +35,10 @@ export function escapeLabelValueInExactSelector(labelValue: string): string {
   return labelValue.replace(/\\/g, '\\\\').replace(/\n/g, '\\n').replace(/"/g, '\\"');
 }
 
+export function unescapeLabelValue(labelValue: string): string {
+  return labelValue.replace(/\\n/g, '\n').replace(/\\"/g, '"').replace(/\\\\/g, '\\');
```

Comment:
Why do we care about `\n` here?

Reply:
Basically I am just blindly doing the opposite as https://github.com/grafana/grafana/pull/59812/files/18b24952aeceef89f0589ed205bec904bc5d2ad2#diff-5ed6a834fd2847e0c6660d68d3ac7e583c188b5940bb1b1ecfd0f8087c8014dfR34, which  is escaping newlines. 

I am also unsure if a label value will ever contain newlines. Guess that's a good question which can be answered in https://github.com/grafana/grafana/issues/59824.